src/components/app/App.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect, Provider } from 'react-redux';
+import { createStore, bindActionCreators } from 'redux';
+import { jumpTo, handleClick } from '../../store/actions';
+import rootReducer from '../../store/reducers';
+import Game from '../game';
+
+const store = createStore(rootReducer);
+
+const putStateToProps = (state) => {
+    return {
+        history: state.history,
+        xIsNext: state.xIsNext,
+        stepNumber: state.stepNumber
+    }
+}
+
+const putActionsToProps = (dispatch) => {
+    return {
+        jumpTo: bindActionCreators(jumpTo, dispatch),
+        handleClick: bindActionCreators(handleClick, dispatch),
+    }
+}
+
+const WrappedGameComponent = connect(putStateToProps, putActionsToProps)(Game)
+
+export default class App extends React.Component {
```

Comment:
Please, use named exports this and further. This is safer and improve working in IDE.